tools/graph-convert/graph-properties-convert-mongodb.cpp

```diff
@@ -1157,6 +1157,6 @@ katana::ConvertMongoDB(
   if (auto r = builder.Finish(); !r) {
     KATANA_LOG_FATAL("Failed to construct graph: {}", r.error());
   } else {
-    return r.value();
+    return std::move(r.value());
```

Comment:
I'm surprised you don't get a warning about an unnecessary move here.

Reply:
Because r.value() does not return the r-value reference. Compiler throws an error without std::move()